pkgs/applications/misc/feedbackd/default.nix

```diff
@@ -0,0 +1,51 @@
+{ stdenv
+, fetchFromGitLab
+, meson
+, ninja
+, pkg-config
+, wrapGAppsHook
+, glib
+, gsound
+, libgudev
+, json-glib
+, vala
+, gobject-introspection
+}:
+
+stdenv.mkDerivation rec {
+  pname = "feedbackd";
+  version = "0.0.0+git20201114";
+
+  src = fetchFromGitLab {
+    domain = "source.puri.sm";
+    owner = "Librem5";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0iv0rf9b5ld2jv2h3cm9l4jvbcm6yjcy93ss6pyb4g3ahpdcdy7k";
+  };
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    pkg-config
+    wrapGAppsHook
+    vala
+    gobject-introspection
+  ];
+
+  buildInputs = [
+    glib
+    gsound
+    libgudev
+    json-glib
+  ];
+
+  meta = with stdenv.lib; {
+    description = "A daemon to provide haptic (and later more) feedback on events";
+    homepage = "https://source.puri.sm/Librem5/feedbackd";
+    license = licenses.gpl3Plus;
+    maintainers = with maintainers; [ masipcat pacman99 ];
```

Comment:
Please add yourself to maintainers.nix.

Reply:
I'm in the maintainer-list.nix, https://github.com/NixOS/nixpkgs/blob/master/maintainers/maintainer-list.nix#L6734

Reply:
Not sure why ofborg failed with this error. Maybe you where not at eval time.

Reply:
Oh I just realized masipcat isn't yet on the list, I removed masipcat for now.